lib/elements/footing.rb

```diff
@@ -0,0 +1,35 @@
+module Elements
+  class Footing
+    attr_reader :concrete
+
+    attr_accessor :step, :dimension1, :dimension2, :height
+
+    def initialize(dimension1:, dimension2:, height:, concrete:, step: 100)
+      @dimension1 = dimension1.to_f
+      @dimension2 = dimension2.to_f
```

Comment:
Just to be clear that these are horizontal dimensions (like ancho y largo), what do you think about calling them `length_x` and `length_y`?

Reply:
done!